src/app/about/index.html

```diff
@@ -1 +1,13 @@
-<my-about></my-about>
+<app-nav role="navigation">
```

Comment:
For non-Stache sites, I think we should aim to put as much markup in the components themselves rather than in the `index.html` file because then it can be unit tested by simply instantiating the component.  Mind making the change to move the fluid grid markup to the respective components so that the contents of the `index.html` files remain simply `<my-component></my-component>`?